notebooks/108-gpu-device/108-gpu-device.ipynb

```diff
@@ -0,0 +1,995 @@
+{
```

Comment:
<p>I believe we should be promoting the MO Ptyhon API from Python code like this...</p><p><a href="https://docs.openvino.ai/latest/openvino_docs_MO_DG_Python_API.html" target="_blank">https://docs.openvino.ai/latest/openvino_docs_MO_DG_Python_API.html</a></p><p> </p><p>At some point there was some work on an OMZ model downloading Python API as well, but that never got merged: <a href="https://github.com/openvinotoolkit/open_model_zoo/pull/1352" target="_blank">https://github.com/openvinotoolkit/open_model_zoo/pull/1352</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
@eaidova was thinking the same, as we shall update other notebooks so the formatting and style will be similar.

Reply:
yes, I think it is my responsibility as notebooks maintainers to update on new API, I plan to make it after the merge. for now, let's focus on GPU not to make the tutorial overcomplicated with non-device-related stuff :)